common/scripted_character_templates/wc_mogu_templates.txt

```diff
@@ -0,0 +1,156 @@
+	after_creation = {
+		add_character_flag = {
+			flag = need_priest_outfit
+		}
+	}
+}
+
+lei_shen_template = { # Lei Shen, the Thunder King
+	age = 550
+	name = Lei_Shen_2
+	gender = male
+	culture = culture:mogu 
+	faith = faith:thunder_king
+	martial=11 diplomacy=4 stewardship=6 intrigue=9 learning=8 prowess = 8
+	trait=education_martial_5 trait=intellect_good_3 trait = physique_good_3 trait = strong trait = thunder_king trait = being_order trait = magic_good_3
+	trait=ambitious trait=arrogant trait=callous
+	trait=creature_mogu
+	random_traits = no
+	after_creation = {
+		really_make_important_lore_character_effect = yes
+		copy_inheritable_appearance_from = character:lineofthunderking1
+		set_house = character:lineofthunderking1.house
+		add_prestige_level = 5
+		add_piety_level = 2
+		give_nickname = nick_the_thunder_king
+	}
+}
+
+warlord_kao = { # Warlord Kao, favored general of Lei Shen
+	age = 134
+	name = Kao
+	gender = male
+	culture = culture:mogu 
+	faith = faith:thunder_king
+	martial=8 prowess = 9
+	trait=education_martial_5
+	trait=aggressive_attacker trait = organizer trait = physique_good_2 trait = loyal trait = being_order
+	trait=wrathful trait=brave trait=callous
+	trait=creature_mogu
+	random_traits = no
+	after_creation = {
+		give_nickname = nick_the_warlord
+		
+		add_trait = lifestyle_blademaster # noted duelist
+		add_trait_xp = {
+			trait = lifestyle_blademaster
+			value = {
+				integer_range = {
+					min = medium_lifestyle_random_xp_low
+					max = medium_lifestyle_random_xp_high
+				}
+			}
+		}
+	}
+}
+
+iron_qon = { # Iron Qon, general of Lei Shen
+	age = 127
+	name = Qon
+	gender = male
+	culture = culture:mogu 
+	faith = faith:thunder_king
+	martial=9 prowess = 9
+	trait=education_martial_3
+	trait=military_engineer trait = unyielding_defender trait = strong trait = being_order
+	trait=diligent trait=brave trait=sadistic
+	trait=creature_mogu
+	random_traits = no
+	after_creation = {
+		give_nickname = nick_iron_qon
+		
+		add_trait = lifestyle_blademaster # noted duelist
+		add_trait_xp = {
+			trait = lifestyle_blademaster
+			value = {
+				integer_range = {
+					min = medium_lifestyle_random_xp_low
+					max = medium_lifestyle_random_xp_high
+				}
+			}
+		}
+	}
+}
+
+jin_rokh_the_breaker = { # Jin'rokh the Breaker, Zandalari dire troll in service of the Thunder King
+	age = 32
+	name = Jin_Rokh
+	gender = male
+	culture = culture:zandalari 
+	faith = faith:zandism
+	martial=8 prowess = 10
+	trait=education_martial_2
+	trait = dire trait = physique_good_2
+	trait=wrathful trait=brave trait=callous
+	trait=creature_troll
+	random_traits = no
+	after_creation = {
+		give_nickname = nick_the_breaker
+		
+		add_trait = lifestyle_blademaster # noted duelist
+		add_trait_xp = {
+			trait = lifestyle_blademaster
+			value = {
+				integer_range = {
+					min = medium_lifestyle_random_xp_low
+					max = medium_lifestyle_random_xp_high
+				}
+			}
+		}
+	}
+}
+
+twin_constort_suen = { # Twin Empyrean Suen
```

Comment:
minor spalling mistake, should be `twin_consort`

Reply:
Fixed!